### ISSUE TITLE
Stay tuned for bug fixes!

### ISSUE THREAD

**Issue by shidiqi007** [apache/rocketmq-dashboard#243]
Can vulnerabilities about this program be fixed, such as CVE-2024-22243

**Comment by princegarg001** [comment]
yes it is necessary to update  addresses CVE-2024-22243, a vulnerability in the Spring Framework that allows open redirect and server-side request forgery (SSRF) attacks when handling external URLs using UriComponentsBuilder.

Fix Implemented:
Upgraded Spring Framework to the latest patched version (6.1.4, 6.0.17, or 5.3.32).
Added strict host validation to prevent untrusted redirects.
Improved URL parsing logic to mitigate SSRF risks.

This fix ensures secure URL handling, preventing attackers from redirecting users to malicious sites or exploiting SSRF vulnerabilities.

Testing & Verification:
Run security scans to confirm the fix.
Tested URL processing functions to ensure expected behavior.

This will definately fixed the vulnerbilty
